SWIPs/swip-bzz-handshake.md

```diff
@@ -0,0 +1,78 @@
+bzz
+===
+
+The bzz protocol consists of a single message type; handshake.
+
+The bzz handshake is the first message to be exchanged between peers
+after the p2p transport protocol has been successfully negotiated. After
+the handshake each peer should remember the following data about the
+other:
+
+-   Swarm Overlay Address of the peer
+
+-   Whether the peer is a *light* or *full* node
+
+message
+-------
+
+The handshake message consists of four data fields:
+
+``` {numbers="none"}
+version     = UINT64
```

Comment:
Indent seems off, also is it "Network ID" or something more like "NetworkID" or "Network_ID"? Inconsistent casing (Address data vs Light node)

Reply:
These identifiers aren't part of the serialization, they are descriptive. Should we enforce a formatting of descriptive fields; no spaces, camelcase? I'm open for that.

`NetworkID` and `Version` should probably also be defined types?

```
VERSION = UINT64
NETWORKID = UINT64

Version = VERSION
Network ID = NETWORKID
... 
```